data_diff/database.py

```diff
@@ -52,18 +54,18 @@ def _one(seq):
     return x
 
 
+def _query_conn(conn, sql_code: str) -> list:
+    c = conn.cursor()
+    c.execute(sql_code)
+    return c.fetchall()

```

Comment:
Why is this not on the `ThreadedDatabase`? 👂🏻

Reply:
Ahh... it's because it's not threaded. If this is the normal pattern rather than instantiating a `SingleThreaded` or just with 1 thread, all good